scripts/zones/Aydeewa_Subterrane/mobs/Chigre.lua

```diff
@@ -0,0 +1,16 @@
+-----------------------------------
+-- Area: Aydeewa Subterrane
+--  ZNM: Chigre
+-----------------------------------
+mixins = {require("scripts/mixins/rage")}
+require("scripts/globals/status")
+-----------------------------------
+-- Todo: add enailments, Drain samba on target if all ailments on, very fast enmity decay
+function onMobInitialize(mob)
+    mob:setMobMod(dsp.mobMod.IDLE_DESPAWN, 300)
+    mob:setLocalVar("[rage]timer", 3600) -- 60 minutes
+    mob:addMod(dsp.mod.MOVE,12)
```

Comment:
this mod will be removed on spawn too, so this addMod should go in onMobSpawn

Reply:
mods applied during init should stay in effect, that behavior where they get wiped is not intended

Reply:
p.s.
if it has to be on onspawn, use setMod instead of addMod to just set it to the exact value it should be.

Reply:
in this case we should cap its speed from retail and change the sql line instead, if possible